protocol.html

```diff
@@ -717,6 +719,51 @@ <h4 property="schema:name"><span>Note</span>: Conditional Update</h4>
+
+                    <p>An <em>N3 Patch</em> is a document in the <em>Notation3 (N3)</em> format [<cite><a class="bibref" href="#bib-notation3">N3</a></cite>], identified by the media type <code>text/n3</code>, conforming to the following structural constraints:</p>
+
+                    <ul>
+                      <li>The patch document can contain patch resources identified by a URI <code>?patch</code> which can be chosen freely.</li>
+                      <li>A patch resource may contain a triple <code>?patch rdf:type solid:Patch</code>.</li>
+                      <li>A patch resource contains at most one triple of the form <code>?patch solid:deletes ?deletions</code>.</li>
+                      <li>A patch resource contains at most one triple of the form <code>?patch solid:inserts ?insertions</code>.</li>
+                      <li>A patch resource contains at most one triple of the form <code>?patch solid:where ?conditions</code>.</li>
+                      <li>When present, <code>?deletions</code>, <code>?insertions</code>, and <code>?conditions</code> must be non-nested <em>cited formulae</em> [<cite><a class="bibref" href="#bib-notation3">N3</a></cite>] consisting only of triples and/or triple patterns. When not present, they are presumed to be the empty formula <code>{}</code>.</li>
+                    </ul>
+
+                    <p>While other specifications might provide a structure and interpretation for a wider class of N3 Patch documents, the present specification only governs the application of N3 Patch documents that additionally adhere to the following structural constraints:</p>
+
+                    <ul>
+                      <li>The patch document contains exactly one patch resource, identified by one or more of the triple patterns described above, which should all share the same <code>?patch</code> subject.</li>
+                      <li>The patch resource may optionally contain one triple of the form <code>?patch solid:matchingStrategy solid:SingleMatch</code>.</li>
+                      <li>The <code>?insertions</code> and <code>?deletions</code> formulae must not contain variables that do not occur in the <code>?conditions</code> formula.</li>
+                      <li>The <code>?insertions</code> and <code>?deletions</code> formulae must not contain blank nodes.</li>
+                      <li>The patch document does not contain any other triples.</li>
```

Comment:
Some of the same here, cap the compliance keywords.

Reply:
In 1d336ff11f56b9689c1a325114a43e94702c38e1